docs/self-managed/identity/user-guide/configuration/connect-to-an-existing-keycloak.md

```diff
@@ -7,12 +7,16 @@ description: "Learn how to connect Identity to your existing Keycloak instance."
 
 In this guide, we'll demonstrate how to connect Identity to your existing Keycloak instance.
 
-### Prerequisites
+## Prerequisites
 
 - Access to your [Keycloak Admin Console](https://www.keycloak.org/docs/22.0.1/server_admin/#using-the-admin-console)
 - A basic understanding of [administering realms and clients](https://www.keycloak.org/docs/22.0.1/server_admin/#assembly-managing-clients_server_administration_guide) in Keycloak.
 
-### Steps
+:::note
+Clients in Camunda 8 SaaS and Camunda 8 Self-Managed are not the same. For Camunda 8 SaaS, you can set up specific [client connection credentials](/docs/guides/setup-client-connection-credentials.md), whereas a client in Identity represents clients which can request [Keycloak to authenticate a user](https://www.keycloak.org/docs/latest/server_admin/).
```

Comment:
@dlavrenuek Maybe you can assist me with the wording here base don your comment...

> Clients in SM have a wider scope than on SaaS but are otherwise pretty similar, on SM we also use the naming Applications. It might be easier to distinguish by that name but on the other hand it should be also clear that both (applications and clients) are used in a similar way for service workers, may be just add a cross reference?

Reply:
Hey @christinaausley - I will try to offer an alternative here :) 

```suggestion
Clients in Camunda 8 SaaS and Applications Camunda 8 Self-Managed provide a similar purpose. One key difference is that for Camunda 8 SaaS, you can set up specific [client connection credentials](/docs/guides/setup-client-connection-credentials.md), whereas in Identity, an Application is created with a credentials automatically assigned.
```